### ISSUE TITLE
Explosions breaking glass cause crash

### ISSUE THREAD

**Issue by brianmta** [multitheftauto/mtasa-blue#3612]
### Describe the bug

Explosions created by createExplosion which break breakable glass objects (such as 3859, 3851) cause crash.

### Steps to reproduce

1. Play a specific map with createExplosion script in it which explodes near glass (code example provided below)
2. Be nearby the explosion, but you don't need to see it happen.

### Version

Client: Multi Theft Auto v1.6-release-22639
OP: Windows 10 Pro 64-bit

### Additional context

CODE EXAMPLE TO BE USED IN A MAP:

local x, y, z = 2485.753, -1667.284, 13.344
createObject(3851,x,y,z)
setTimer(function()
    createExplosion( x,y,z, 7)
end,100,1)

### Relevant log output

```shell
Version = 1.6-release-22639.0.000
Time = Sun Jul 28 13:59:33 2024
Module = C:\Program Files (x86)\MTA San Andreas 1.6\mta\multiplayer_sa.dll
Code = 0xC0000005
Offset = 0x00004DD2

EAX=0071C28E  EBX=0000006D  ECX=0177F120  EDX=1FFCAB30  ESI=C4D06917
EDI=0177F448  EBP=00000000  ESP=0177F114  EIP=69334DD2  FLG=00010246
CS=0023   DS=002B  SS=002B  ES=002B   FS=0053  GS=002B
```

### Security Policy

- [X] I have read and understood the Security Policy and this issue is not security related.

**Comment by TracerDS** [comment]
Related info:
![image](https://github.com/user-attachments/assets/cc16c873-1967-4cc4-883d-caf23ebad647)
![image](https://github.com/user-attachments/assets/4536852a-681c-4f11-bde9-e5154373e74a)
![image](https://github.com/user-attachments/assets/6ef93efe-526d-4e2e-bce2-42eea3ed669a)

- `multiplayer_sa_d.dll!HOOK_CGlass_WindowRespondsToCollision() Line 6839`
- `Client/multiplayer_sa/CMultiplayerSA.cpp:6839`

TLDR; ~~Something is wrong with sounds causing the crash to occur.~~

`ebp` is 0. Registers are corrupted
![image](https://github.com/user-attachments/assets/b1bd5fc0-b020-4a53-9cdd-61635a36be71)

**Comment by TheNormalnij** [comment]
Is  it related to #3533, @FileEX ?

**Comment by FileEX** [comment]
Yes, I'm working on it

**Comment by FileEX** [comment]
> Related info: ![image](https://private-user-images.githubusercontent.com/43095317/352811586-cc16c873-1967-4cc4-883d-caf23ebad647.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxNzAwMDEsIm5iZiI6MTcyMjE2OTcwMSwicGF0aCI6Ii80MzA5NTMxNy8zNTI4MTE1ODYtY2MxNmM4NzMtMTk2Ny00Y2M0LTg4M2QtY2FmMjNlYmFkNjQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI4VDEyMjgyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczZGNhZTcxNDlhOGVhOWU0MWI4NTJlZDg5OTk1OTRlNGEzYzVkY2VlNDAzYzM3MzFjNDI5MDYyZjIwMmNjOWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.INHsLzB-KRWSA0AAaRwL_QrX7vceIiBa1HWAXdZlp1o) ![image](https://private-user-images.githubusercontent.com/43095317/352811644-4536852a-681c-4f11-bde9-e5154373e74a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxNzAwMDEsIm5iZiI6MTcyMjE2OTcwMSwicGF0aCI6Ii80MzA5NTMxNy8zNTI4MTE2NDQtNDUzNjg1MmEtNjgxYy00ZjExLWJkZTktZTUxNTQzNzNlNzRhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI4VDEyMjgyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTdjZGRhOTAxZjRmMDUwODczYTQzMzJjYzM3OGYzZWFlY2RmMmVlZmQyZmEyZjBiOTgzZWUyYjY0NTJmZGY0Y2QmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8oOI-bWiLJvSv-WQjGerumL_QkEaWKHHhJB7Sib3V6U) ![image](https://private-user-images.githubusercontent.com/43095317/352811678-6ef93efe-526d-4e2e-bce2-42eea3ed669a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxNzAwMDEsIm5iZiI6MTcyMjE2OTcwMSwicGF0aCI6Ii80MzA5NTMxNy8zNTI4MTE2NzgtNmVmOTNlZmUtNTI2ZC00ZTJlLWJjZTItNDJlZWEzZWQ2NjlhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI4VDEyMjgyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE0MmNiNDBhZjViNzNhODNmNmZhOTA0ODJkNzQwOGQ2YTE5YTZhMWZjMTlmMWFiMTMyOTMxOGI4MTM4NWNlMWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CEv-G66X9yKnJSL-ep_m5yK7wni7sEv7dfJpiCu46Vk)
> 
> * `multiplayer_sa_d.dll!HOOK_CGlass_WindowRespondsToCollision() Line 6839`
> * `Client/multiplayer_sa/CMultiplayerSA.cpp:6839`
> 
> TLDR; Something is wrong with sounds causing the crash to occur.
> 
> `ebp` is 0. Registers are corrupted ![image](https://private-user-images.githubusercontent.com/43095317/352811865-b1bd5fc0-b020-4a53-9cdd-61635a36be71.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxNzAwMDEsIm5iZiI6MTcyMjE2OTcwMSwicGF0aCI6Ii80MzA5NTMxNy8zNTI4MTE4NjUtYjFiZDVmYzAtYjAyMC00YTUzLTljZGQtNjE2MzVhMzZiZTcxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI4VDEyMjgyMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQxNTkwYjhhOWM5MWY2MDdlNjk4NDZiMWZhYmExZmNiZjNlYWIwMjE4MTdhMzFlYWNhZWJhOGM2MWFlMTZjMzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3LPHltawAcV57cDQvYJ-kcyALvHYLhmHdmMWaSUfSkg)

Crash is not related to sounds
